loom/common/utils/utByteArray.h

```diff
@@ -32,6 +32,16 @@ class utByteArray {
     utArray<unsigned char> _data;
     UTsize _position;
 
+    void memcpyAligned(void *destination, const void *source, size_t num)
+    {
+        // TODO smarter
+        // Loop copying byte by byte to avoid unaligned access
+        for (size_t i = 0; i < num; i++)
+        {
+            memcpy((void*)(&((unsigned char*)destination)[i]), (const void*)(&((const unsigned char*)source)[i]), 1);
```

Comment:
I'll have to test this on all platforms again. And I think it's without the * as array access already implies it :)